docs/tutorials/time-based-benchmarking.md

```diff
@@ -0,0 +1,182 @@
+<!--
+SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Time-based Benchmarking
+
+Time-based benchmarking allows you to run benchmarks for a specific duration rather than a fixed number of requests.
+This approach is ideal for measuring sustained performance and testing service stability over time.
+
+## Overview
+
+Time-based benchmarking provides several advantages:
+
+- **Consistent Measurement Window**: Compare performance across different configurations using the same time duration
+- **Real-world Simulation**: Mirror production scenarios where load is sustained over time
+- **Resource Utilization**: Identify memory leaks, connection pooling issues, and resource exhaustion patterns
+- **SLA Validation**: Establish and verify performance guarantees over specific time periods
+- **Grace Period Control**: Handle in-flight requests gracefully or force immediate completion as needed
+
+## Core Parameters
+
+### Benchmark Duration
+- `--benchmark-duration SECONDS`: Total time to run the benchmark
+- Overrides `--request-count` when specified
```

Comment:
i think it errors out right?

Reply:
They are mutually exclusive. I'll update the wording in case it's confusing.